handler.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"cloud.google.com/go/logging/jsonlog"
+)
+
+func (a *App) Handler(w http.ResponseWriter, r *http.Request) {
+	l, err := jsonlog.NewLogger(fmt.Sprintf("projects/%s", a.projectID))
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	l.WithRequest(r).
```

Comment:
Maybe pull all of this into a method on `*App`? Something like `a.Logger(r)` and under the hood that creates a logger for this request?